mne/gui/__init__.py

```diff
@@ -201,17 +201,19 @@ def coregistration(tabbed=False, split=True, width=None, inst=None,
 
 
 @verbose
-def locate_ieeg(info, trans, aligned_ct, subject=None, subjects_dir=None,
+def locate_ieeg(info, trans, base_image, subject=None, subjects_dir=None,
```

Comment:
this is technically a silent API change

up to you...

Reply:
It would be nice not to have it be a misnomer, I could depreciate it properly though... I put a note in latest